docs/conf.py

```diff
@@ -107,6 +107,7 @@ def check_if_v046():
 # further.  For a list of options available for each theme, see the
 # documentation.
 html_theme_options = {
+    "logo_link": "index",
```

Comment:
I noticed this TODO comment - shall we link the logo to one of these links instead?